css/style.css

```diff
@@ -283,10 +289,68 @@ footer ul li {
   margin: 0 26px 75px;
 }
 footer p {
-  font-size: 18px;
+  font-size: 0.692rem;
   color: #fff;
   font-weight: 600;
 }
 
 
 
+/* Mobile */
+
+@media screen and (max-width: 768px) {
+
+  .container {
+    width: 90%;
+  }
+
+  blockquote {
+    width: 90%;
+  }
+
+  #benefits, #benefits p, #benefits ul {
+    width: 90%;
+    font-size: 1rem;
+  }
+
+  html {
+    font-size: 16px;
+  }
+
+  #hero h2 {
+    margin: 20px 0 20px;
+  }
+
+  #hero {
+    height: 250px;
```

Comment:
Make this `min-height: 250px` so that way it can grow if the height needs to be bigger